plugins/main/public/controllers/management/components/management/cdblists/views/list-editor.tsx

```diff
@@ -259,6 +314,15 @@ class WzListEditor extends Component {
       );
       return;
     }
+    if (this.state.isInvalid.length > 0) {
+      this.showToast(
+        'danger',
+        'Error',
+        'Key and value must start and end with quotes or have no quotes at all',
```

Comment:
Utilize the previously created constant and convert its value to lowercase.

Reply:
Done [310743b](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7171/commits/310743b1942f73af2d2e2df3efd88ba0be294136)